tools/docker/README.md

```diff
@@ -17,12 +17,12 @@ If `build.sh` or `run.sh` are called without first calling `image-build.sh`, the
 
 ## Docker builder
 
-The docker builder image can be used to build prplMesh via `maptools.py build` command running in a prplmesh-build container via `build.sh`.
```

Comment:
builder docker should remain only for build. You have runner docker for anything else.